eventsource-channel-2023.html

```diff
@@ -0,0 +1,490 @@
+                <div datatype="rdf:HTML" property="schema:description">
+                  <p><em>This section is non-normative.</em></p>
+
+                  <p>The following diagram shows the high-level interactions involved in this flow. How a client retrieves an access token for step 5 is outside the scope of this document.</p>
+
+                  <figure id="solid-eventsource-flow" rel="schema:hasPart" resource="#solid-eventsource-flow">
+                    <img src="eventsource-channel-2023-flow.mmd.svg" rel="schema:image" width="800" />
+
+                    <figcaption property="schema:name">Solid EventSourceChannel2023 Flow</figcaption>
+                  </figure>
+                </div>
+              </section>
+
+              <section id="conformance" inlist="" rel="schema:hasPart" resource="#conformance">
+                <h3 property="schema:name">Conformance</h3>
+                <div datatype="rdf:HTML" property="schema:description">
+                  <p>All assertions, diagrams, examples, and notes are non-normative, as are all sections explicitly marked non-normative. Everything else is normative.</p>
+
+                  <p>The key words “MUST” and “MUST NOT” are to be interpreted as described in <cite><a href="https://tools.ietf.org/html/bcp14" rel="rdfs:seeAlso">BCP 14</a></cite> [<cite><a class="bibref" href="#bib-rfc2119">RFC2119</a></cite>] [<cite><a class="bibref" href="#bib-rfc8174">RFC8174</a></cite>] when, and only when, they appear in all capitals, as shown here.</p>
+                </div>
+              </section>
+            </div>
+          </section>
+
+          <section id="channel-type" inlist="" rel="schema:hasPart" resource="#channel-type">
+            <h2 property="schema:name">EventSourceChannel2023 Type</h2>
+            <div datatype="rdf:HTML" property="schema:description">
+              <p>This specification defines the <code>EventSourceChannel2023</code> type for use with Solid Notifications subscriptions that use the EventSource Web API. The URI of the channel type is <code>http://www.w3.org/ns/solid/notification#EventSourceChannel2023</code>.</p>
+              <p>An <code>EventSourceChannel2023</code> API MUST conform to the <cite><a href="https://solid.github.io/notifications/protocol#discovery">Solid Notifications Protocol</a></cite>. [<cite><a class="bibref" href="#bib-solid-notifications">SOLID-NOTIFICATIONS</a></cite>]</p>
+
+              <p>An <code>EventSourceChannel2023</code> API SHOULD support the <cite><a href="https://solid.github.io/notifications/protocol#notification-features">Solid Notifications Features</a></cite>.</p>
+
+              <p>A client establishes a subscription using the <code>EventSourceChannel2023</code> type by sending an authenticated subscription request to the hypermedia API retrieved via <cite><a href="https://solid.github.io/notifications/protocol#discovery">Solid Notifications discovery</a></cite>.</p>
```

Comment:
"to the hypermedia API retrieved via" is unclear to me or perhaps it is outdated now since the terminology in Solid Notifications Protocol changed. Perhaps refer to "subscription resource" (or service/inbox or whatever we are going with..) instead. The paragraph may need to be adjusted a bit more.

```suggestion
              <p>A client establishes a subscription using the <code>EventSourceChannel2023</code> type by sending an authenticated subscription request to the subscription resource based on <cite><a href="https://solid.github.io/notifications/protocol#discovery">Solid Notifications Protocol discovery</a></cite>.</p>
```